### DISCUSSION TITLE
Just found some typo error on documentation . Please check if it wrong or my understanding itself is wrong

### DISCUSSION THREAD

**Discussion by shivamkrsingh** [cap-js/.github#4] [General]
![image](https://github.com/cap-js/.github/assets/14155873/afd36c03-1df9-49b7-b8ac-0e08df810ede)

Just found some typo error on documentation . managed  aspect can not be used without mentioning it as aspect
using { Country } from '@sap/cds/common';
entity Addresses : managed { //> **using reuse aspect**
  street  : String;
  town    : String;
  country : Country; //> using reuse type
}

**Comment by chgeo** [comment]
Thanks. As this is rather an issue than a discussion and relates to cap-bs/docs, I have created an issue there: https://github.com/cap-js/docs/issues/692
Please subscribe there to track progress.
